packages/ui/src/lib/adapters/zkitter/index.ts

```diff
@@ -588,8 +562,24 @@ export class ZkitterAdapter implements Adapter {
 			ecdsa,
 		}))
 
-		await this.syncChats()
+		const contract = getGlobalAnonymousFeed()
+
+		const { IndexedDBConnector } = await import('anondb/web');
+
+		const db = await IndexedDBConnector.create(schema)
+		this.userState = new UserState(
```

Comment:
this caches userState in memory whenever `signIn` is called so that it doesn't try to sync with unirep every time you make a call